src/Language/PureScript/Interactive.hs

```diff
@@ -217,12 +226,15 @@ handleImport
   => ImportedModule
   -> m ()
 handleImport im = do
-   st <- gets (updateImportedModules (im :))
+   st <- gets (updateImportedModules (renew im))
    let m = createTemporaryModuleForImports st
    e <- liftIO . runMake $ rebuild (map snd (psciLoadedExterns st)) m
    case e of
      Left errs -> printErrors errs
      Right _  -> put st
+   where
+   renew :: ImportedModule -> [ImportedModule] -> [ImportedModule]
```

Comment:
I think if we were going to do that, we should consolidate any imports under the same namespace. So for example

```
import Prelude ((+))
import Prelude (bind)
import Prelude as P ((*))
import Prelude as P (show)
```

should give

```
> :show import
import Prelude ((+), (*))
import Prelude as P (bind, show)
```

Without that, I'm not sure it's worth it to be honest.